src/server.h

```diff
@@ -1992,6 +1992,8 @@ struct redisServer {
     /* Latency monitor */
     long long latency_monitor_threshold;
     dict *latency_events;
+    long long el_start;
+    dict *duration_stats;
```

Comment:
unlike the latency monitor, which is an infrastructure that has a ton of sample points and a generic command to extract them, here we have some 6 different metrics, hard wired to the INFO command.
i'm not sure we want to have the overhead of dict lookup, it would be faster to just directly access dedicated server struct variables.